railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -20,6 +20,13 @@ def edit
         require_application_and_environment!
 
         ensure_editor_available(command: "bin/rails credentials:edit") || (return)
+
+        if File.exist?("config/credentials.yml.enc") && !File.exist?("config/master.key")
```

Comment:
This check does not work properly when `ENV["RAILS_MASTER_KEY"]` is used. 
When `ENV["RAILS_MASTER_KEY"]` is used, there is no `config/master.key` but data can be decrypted.

Reply:
Thanks, I will add the condition.